src/OnboardingSPA/components/Sidebar/components/Customize/DesignFontsPanel/index.js

```diff
@@ -191,8 +191,13 @@ const DesignFontsPanel = forwardRef(
 		const resetToDefaultFonts = () => {
 			setStylesOfCurrentData();
 			setSelectedGroup( null );
-			setShowCustomFonts( false );
-			setSelectedCustomFont( null );
+			const slug = currentData.sitegen?.homepages?.active?.slug;
```

Comment:
Can we add a `useEffect` in the parent component so that whenever `currentData.sitegen.homepages.active` changes, it automatically makes a change in the corresponding data in `currentData.sitegen.homepages.data`?